editOrder.php

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="scripts/editorder.js"></script>
+    <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Edit Order</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="editorder" id="editorder">
+        <div>
+            <a class="back-item" href="orderSummery.php">
+                <?php echo file_get_contents("./assets/icons/FiArrowLeft.svg"); ?> Go Back
+            </a>
```

Comment:
May want to double check the wording in Figma.

Reply:
Changed it ti Order Summary